diff --git a/node_modules/@svelte-put/modal/dist/Modal.svelte b/node_modules/@svelte-put/modal/dist/Modal.svelte
index 8c53df6..78a9bfc 100644
--- a/node_modules/@svelte-put/modal/dist/Modal.svelte
+++ b/node_modules/@svelte-put/modal/dist/Modal.svelte
@@ -82,6 +82,7 @@ function onPressEscape() {
 <slot name="backdrop" class={rClasses.backdrop} onClick={onClickBackdrop}>
   {#if backdrop}
     <!-- svelte-ignore a11y-click-events-have-key-events -->
+    <!-- svelte-ignore a11y-no-static-element-interactions -->
     <div class={rClasses.backdrop} on:click={onClickBackdrop} transition:fade={{ duration: 75 }} />
   {/if}
 </slot>
